tests/phpunit/Unit/Components/NavMenuTest.php

```diff
@@ -41,4 +43,38 @@ class NavMenuTest extends GenericComponentTestCase {
 
 	protected $classUnderTest = '\Skins\Chameleon\Components\NavMenu';
 
+	/**
+	 * @covers ::getHTML
+	 * @dataProvider domElementProviderFromSyntheticLayoutFiles
+	 */
+	public function testGetHTML_HasValidId( $domElement ) {
+		$chameleonTemplate = $this->getMockBuilder( ChameleonTemplate::class )
```

Comment:
This is no longer needed in modern PHPUnit

Just do `$this->createMock( ChameleonTemplate::class );`